src/main/java/net/minecraftforge/event/server/ServerLifecycleEvent.java

```diff
@@ -0,0 +1,87 @@
+package net.minecraftforge.event.server;
+
+import net.minecraft.server.MinecraftServer;
+import net.minecraftforge.common.MinecraftForge;
+import net.minecraftforge.fml.common.eventhandler.Cancelable;
+import net.minecraftforge.fml.common.eventhandler.Event;
+
+public class ServerLifecycleEvent extends Event
+{
+    private final MinecraftServer server;
+    protected ServerLifecycleEvent(MinecraftServer server)
+    {
+        this.server = server;
+    }
+    public MinecraftServer getServer()
+    {
+        return this.server;
+    }
+    
+    /**
+     * This event is fired before the server starts.
+     * <br>
+     * This event is not {@link Cancelable}.<br>
+     * <br>
```

Comment:
There's a `<br>` on the previous line - are two needed? (This applies to all cases of this.)